mmv1/products/identityplatform/Config.yaml

```diff
@@ -228,20 +228,29 @@ properties:
       - name: 'signUpQuotaConfig'
         type: NestedObject
         description: |
-          Quota for the Signup endpoint, if overwritten. Signup quota is measured in sign ups per project per hour per IP.
+          Quota for the Signup endpoint, if overwritten. Signup quota is measured in sign ups per project per hour per IP. None of the child properties can be skipped.
```

Comment:
Be more explicit in case we add new fields in the future: None of quota, startTime, or quotaDuration can be skipped.

Reply:
Makes sense. Updated